system-int-test/test/cmr/system_int_test/search/collection_doi_search_test.clj

```diff
@@ -50,4 +58,19 @@
        [coll1 coll2] ["Doi*"] {:pattern true}
 
        "search for collections with both doi1 and doi2 returns nothing"
-       [] ["Doi1" "doI2"] {:and true}))))
+       [] ["Doi1" "doI2"] {:and true}))
+    (testing "search doi with json query"
+      (are3 [items json-search]
+            (d/refs-match? items (search/find-refs-with-json-query
+                                  :collection {} json-search))
+            "Search for empty values"
+            [coll3] {:doi {:value "Not provided" :pattern true}}
+
+            "Wildcard search"
+            [coll1 coll2 coll3] {:doi {:value "*" :pattern true}}
+
+            "Search for collections with doi1 or doi2"
+            [coll1 coll2] {:doi {:value "Doi*" :pattern true}}
+
+            "Search for collections with doi1"
```

Comment:
The main purpose of the ticket is to be able to search for collections without DOI values. We should have some collections that do not have a DOI field at all (rather than "Not provided") and add two more tests:

1. `{:not {:doi {:value "*" :pattern true}}}` returns any collections without a DOI
2. `{:or [{:not {:doi {:value "*" :pattern true}}} {:not {:doi "Not provided"}}]` returns any collections without a DOI or with a value of Not provided.

Side note - the tests with Not provided probably don't make sense for DOI, they should be added for another field which is required in UMM JSON.

Reply:
Done